test/mpi/threads/spawn/th_taskmaster.c

```diff
@@ -101,6 +101,10 @@ int main(int argc, char *argv[])
 #endif /* USE_THREADS */
 
 #ifdef USE_THREADS
+        /* Initialize the thread package. It should not be initialized via
+         * MTest_Init_thread since MTest_Finalize cannot be called. */
```

Comment:
It is because both `MTest_Init_thread` and `MTest_Finalize` _are_ not called rather than _cannot_ be called, right?

Reply:
You are right. I will change the comment. Thanks a lot for reviewing this PR!

Reply:
@hzhou I fixed the comment.